src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <name>Microwave Oven Control</name>
+    <domain>Appliances</domain>
+    <description>Attributes and commands for configuring the microwave oven control, and reporting cooking stats.</description>
+    <code>0x050F</code>
+    <define>MICROWAVE_OVEN_CONTROL_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <attribute side="server" code="0x0001" define="COOK_TIME" type="elapsed_s" writable="false" optional="false">CookTime</attribute>
+    <attribute side="server" code="0x0002" define="POWER_SETTING" type="int8u" writable="false" optional="false">PowerSetting</attribute>
+    <attribute side="server" code="0x0003" define="MIN_POWER" type="int8u" writable="false" optional="true">MinPower</attribute>
+    <attribute side="server" code="0x0004" define="MAX_POWER" type="int8u" writable="false" optional="true">MaxPower</attribute>
+    <attribute side="server" code="0x0005" define="POWERSTEP" type="int8u" writable="false" optional="true">PowerStep</attribute>
+
+    <command source="client" code="0x00" name="SetCookingParameters" optional="false">
+        <description>Set Cooking Parameters</description>
+        <arg name="CookMode" type="int8u" optional="true"/>
+        <arg name="CookTime" type="elapsed_s" optional="true"/>
+        <arg name="PowerSetting" type="int8u" min="MIN_POWER" max="MAX_POWER" optional="true"/>
```

Comment:
Pretty sure those min/max values would not be understood if anything were trying to use them, but nothing is right now.